packages/shared/src/components/cards/SharedPostCardFooter.tsx

```diff
@@ -2,39 +2,51 @@ import classNames from 'classnames';
 import React, { ReactElement } from 'react';
 import { Post } from '../../graphql/posts';
 import { cloudinary } from '../../lib/image';
+import { IconSize } from '../Icon';
+import { CardImage, CardVideoImage } from './Card';
 
 type SharedPostCardFooterProps = {
   isShort: boolean;
+  isVideoType?: boolean;
 } & Pick<Post, 'sharedPost'>;
 
 export const SharedPostCardFooter = ({
   sharedPost,
   isShort,
+  isVideoType,
 }: SharedPostCardFooterProps): ReactElement => {
+  const ImageComponent = isVideoType ? CardVideoImage : CardImage;
   return (
     <div
       className={classNames(
-        'flex flex-1 gap-2 p-3 mb-2 rounded-12 border border-theme-divider-tertiary',
+        'p-3 gap-3 mb-2 rounded-12 border border-theme-divider-tertiary flex w-fit max-h-[13.5rem] flex-auto h-auto min-h-[0]',
         isShort ? 'flex-row items-center' : 'flex-col',
       )}
     >
-      <span
+      <div
         className={classNames(
-          'typo-footnote text-theme-label-secondary',
-          isShort ? 'line-clamp-4 flex-1' : 'line-clamp-2',
+          'typo-footnote text-theme-label-secondary flex',
+          isShort ? 'line-clamp-4 pr-3 w-8/12' : 'line-clamp-2',
```

Comment:
Why do we need the w-8/12 here? Feels a bit odd to use.

Reply:
So from design the title takes almost 66% of the container and 33% the image (when there are more then 5 lines of post title text), so used this to make sure. If there is any other way to do this top of your mind, please let me know 🙏

Reply:
And it should apply to all shared post?
I can't really recall

Reply:
Yes, to all shared post whose main post title is more then 5 lines. So when the main title is more then 5 lines (this is what the user writes when they share a post) then the shared post actual title and image should show side by side.
<img width="349" alt="Screenshot 2023-12-12 at 4 10 48 PM" src="https://github.com/dailydotdev/apps/assets/36197304/cd549e01-30d6-403a-b63b-9506eb0cfb47">

Reply:
But this is a existing implementation right?
Just wondering how this worked alright before then?

Reply:
it was not using the image component, it was using a div with the post image as a background image

Reply:
We decided we wanted to unify this in the comments above when the suggestion for VideoImage component was made

Reply:
Oh I see, ok no worries in that case :) 
Alternative would maybe be the image component to have height of parent with square aspect ratio.
Then flex-1 on the text block to fill remaining space?

But also ok with this.
I'm not sure if the above has downsides/issues.

Reply:
I tried and the image takes a a bigger height then the parent, as the parent doesn't have a set height, maybe I'm missing something there. 
But thinking of merging this as is so I don't block the feature, and we can check together later if you want.